### ISSUE TITLE
Diff of data/cohort-data.json still too large

### ISSUE THREAD

**Issue by matentzn** [IHCC-cohorts/data-harmonization#111]
I made a commit to normalise the diff [here](https://github.com/IHCC-cohorts/data-harmonization/commit/c3191b67338d48dd45407a46bf45f5ebf41f4fa2#diff-0859d67cfe8e5b876b2a6057a39022bd).

To not confuse the Data folks, it would be good if we could make sure that changes to cohort-data.json are minimal. (IMO, only one block of green, for the new data dictionary). 

I will be bold and assign @beckyjackson but if you dont want the ticket, fee free to assign it to @jamesaoverton instead :)

**Comment by matentzn** [comment]
Here is a test PR with the diff as well:
https://github.com/IHCC-cohorts/data-harmonization/pull/110/files

**Comment by matentzn** [comment]
(the other json files were fine after normalisation)

**Comment by beckyjackson** [comment]
It looks like the cohort names aren't sorted alphabetically and running the task to rebuild the cohort data ends up with the values sorted alphabetically. I added a cohort and regeneerated the data from master and the diff was a mess. I commited my changes to a test branch then I removed the cohort from `data/metadata.json` and ran `make data/cohort-data.json` again. The second diff looked as-expected:
![image](https://user-images.githubusercontent.com/16750150/95024828-97935700-0653-11eb-8ffb-a6462792ff11.png)
